### DISCUSSION TITLE
Rotate a screen 90º

### DISCUSSION THREAD

**Discussion by plpxsk** [rxhanson/RectanglePro-Community#147] [Ideas]
It's getting to me that, in macOS, it's quite "slow" to rotate my second screen 90º with all the clicking. I need to do this a few times per day.

Are there plans to incorporate screen rotation into this app?

**Comment by rxhanson** [comment]
At the moment, I don't have plans to add this into the app. Rectangle Pro is more about managing windows than displays. I did a quick search to see what was out there that does this, and it looks like available options are broken on the latest Macs/macOS. This means that it would be something fragile to implement if possible, and would take away from being able to add more stable functionality.

**Comment by plpxsk** [comment]
Makes sense. Indeed, looks like best option for this is an ad-hoc apple script or similar...
